src/useLoadBans.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, type Dispatch } from "react";
+import { type Action } from "./useAppState";
+import { normalizeString } from "./Normalization";
+
+export default function useLoadBans(dispatch: Dispatch<Action>): void {
+  useEffect(() => {
+    fetch("https://unpkg.com/naughty-words@1.2.0/en.json").then((response) =>
+      response.json().then((bannedWords) =>
```

Comment:
This probably works but it's not quite the nesting I had in mind

```suggestion
    fetch("https://unpkg.com/naughty-words@1.2.0/en.json").then((response) =>
      response.json()).then((bannedWords) =>
```